### ISSUE TITLE
[BUG] Rate-Limit Banned. This bot is not working currently. It was working fine 4 hrs ago. Check this?

### ISSUE THREAD

**Issue by forthetemp** [rdeepak2002/reddit-place-script-2022#210]
2022-04-03 08:50:57.404 | INFO     | __main__:get_board:219 - Getting board
Authorization successful!
Obtaining access token...
2022-04-03 08:50:59.352 | INFO     | __main__:get_board:219 - Getting board
2022-04-03 08:50:59.373 | INFO     | __main__:get_board:338 - Got image.
2022-04-03 08:51:01.246 | INFO     | __main__:get_board:338 - Got image.
2022-04-03 08:51:09.386 | INFO     | __main__:get_board:219 - Getting board
2022-04-03 08:51:11.259 | INFO     | __main__:get_board:219 - Getting board
2022-04-03 08:51:11.371 | INFO     | __main__:get_board:338 - Got image.
2022-04-03 08:51:11.831 | INFO     | __main__:task:558 - 
Account Placing: 
2022-04-03 08:51:11.832 | INFO     | __main__:set_pixel_and_check_ratelimit:158 - Thread #0 : Attempting to place Black (27) pixel at 1869, 791
2022-04-03 08:51:12.591 | ERROR    | __main__:set_pixel_and_check_ratelimit:200 - Thread #0 : Failed placing pixel: rate limited
2022-04-03 08:51:13.543 | INFO     | __main__:get_board:338 - Got image.
2022-04-03 08:51:13.598 | INFO     | __main__:task:432 - Thread #0 :: CANCELLED :: Rate-Limit Banned
2022-04-03 08:51:14.033 | INFO     | __main__:task:558 - 
Account Placing: 
2022-04-03 08:51:14.033 | INFO     | __main__:set_pixel_and_check_ratelimit:158 - Thread #1 : Attempting to place Black (27) pixel at 1869, 791
2022-04-03 08:51:14.793 | ERROR    | __main__:set_pixel_and_check_ratelimit:200 - Thread #1 : Failed placing pixel: rate limited
2022-04-03 08:51:15.794 | INFO     | __main__:task:432 - Thread #1 :: CANCELLED :: Rate-Limit Banned

**Comment by Plantchant** [comment]
That means your account was banned from placing pixels on reddit. This does not mean it's permanent. Try using VPN or logging in. Or just use new accounts if you're doing massive numbers.

**Comment by forthetemp** [comment]
I am using mutliple accounts. But wait I will try again.

**Comment by forthetemp** [comment]
I tried two other accounts. But for both it said authorization failed.
Please check this.

**Comment by forthetemp** [comment]
Wait you are saying it's running right?. Are you providing the secret key and name of the reddit app?

**Comment by deleniti** [comment]
> That means your account was banned from placing pixels on reddit. This does not mean it's permanent. Try using VPN or logging in. Or just use new accounts if you're doing massive numbers.

Rate limits are permanent. I tried logging into my main account using 2 different VPN locations and am still rate-limited for forever.


> And it is currently working. I'm running 120 accs right now
HOW?

**Comment by forthetemp** [comment]
So no. Again I got rate limit banned. Even though these were new accounts.

**Comment by Nebula6** [comment]
yup i made 20 accounts and they all placed 1 pixel before they all got banned, they cant place a pixel for the next 15 years
![image](https://user-images.githubusercontent.com/62147347/161422343-094df35c-77a7-4f01-a19c-8676868045a7.png)

**Comment by Plantchant** [comment]
Here's my process for making accounts...

> 10minutemail for temporary email.
> Make reddit account and fill out those stupid questions it asks
> Verify email
> Go to r/place and place one pixel

Now the account should be less likely to get banned. This is what I've been doing and it's working with over 100 accs right now.

And also you can have rate-limit bans go away. I don't know how to trigger it but I've had dead accounts rise.

**Comment by Antonio32A** [comment]
> Here's my process for making accounts...
> 
> > 10minutemail for temporary email.
> > Make reddit account and fill out those stupid questions it asks
> > Verify email
> > Go to r/place and place one pixel manually
> 
> Now the account should be less likely to get banned. This is what I've been doing and it's working with over 100 accs right now.
> 
> And also you can have rate-limit bans go away. I don't know how to trigger it but I've had dead accounts rise.

Just wondering, which proxies/VPN are you using? 
Also do you have to swap them between creating accounts and if yes, does Reddit detect that you've created an account using a VPN/proxy and then switched to a different one later?

**Comment by Plantchant** [comment]
> > Here's my process for making accounts...
> > > 10minutemail for temporary email.
> > > Make reddit account and fill out those stupid questions it asks
> > > Verify email
> > > Go to r/place and place one pixel manually
> > 
> > 
> > Now the account should be less likely to get banned. This is what I've been doing and it's working with over 100 accs right now.
> > And also you can have rate-limit bans go away. I don't know how to trigger it but I've had dead accounts rise.
> 
> Just wondering, which proxies/VPN are you using? Also do you have to swap them between creating accounts and if yes, does Reddit detect that you've created an account using a VPN/proxy and then switched to a different one later?

I don't use proxies. I just switch on "HMA" VPN every once in a while. Reddit probably does save that information but so far I haven't seen them care.

**Comment by eichelb4rt** [comment]
> 10minutemail for temporary email.
> Make reddit account and fill out those stupid questions it asks
> Verify email
> Go to r/place and place one pixel manually

just tried that with 4 accounts, signed up on 2 different VPN locations. One of the accounts succeeded once and got banned, the others immediately got banned.

**Comment by Inphinitation** [comment]
I have a bunch of new accounts that all work when logged in manually, but the bot returns that they're rate limit banned
So I think something is actually wrong

**Comment by forthetemp** [comment]
Is the issue resolved yet?

**Comment by michaelshumshum** [comment]
what are the reddit accoutn names you are using? maybe that could be a factor.
